webui/react/src/components/ResourceChart.tsx

```diff
@@ -108,17 +107,14 @@ const SlotChart: React.FC<Props> = (props: Props) => {
   ) => {
     const args: PlotArguments = [ elementId, pInfo.data, pInfo.layout, pInfo.config ];
     await Plotly.react.apply(null, args);
-  }, [ id, plotInfo, props.resources ]);
+  }, [ ]);
 
   useEffect(() => {
     renderPlot(id, plotInfo);
-  }, [ id, plotInfo, props.resources ]);
+  }, [ id, plotInfo, renderPlot ]);
```

Comment:
**chore:** good call with eliminating the `renderPlot` and just moving the `Plotly.react` directly into the `useEffect`